docker-compose.example.yml

```diff
@@ -0,0 +1,99 @@
+version: "3"
+services:
+  # The postgres server
+  database:
+    # Pulling the image in the latest tag is a disaster.
+    image: postgres:13-alpine
+    environment: 
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRE_DB=postgres
+    volumes:
+    - dbdata:/var/lib/postgresql/data
+    ports:
+    - 5432:5432
+  
+  # https://github.com/benawad/dogehouse/issues/101#issuecomment-778605325
+  adminer:
+    image: adminer
+    network_mode: host
+    restart: "no"
+    ports:
+      - 8080
+    depends_on:
+      - database
+  
+  # The WSS server used to establish and manage voice and text rooms
+  voice_server:
+    network_mode: host
+    build:
+      context: ./shawarma
+      dockerfile: Dockerfile
+    depends_on: 
+      - backend
+
+  # The backend server
+  backend:
+    network_mode: host
+    build: 
+      context: ./kousa
+      dockerfile: Dockerfile
+    depends_on: 
+      - database
+    environment:
```

Comment:
Wouldn't it be better to have all these environment variables in a `.env` file and keep things clean?